language-support/hs/bindings/src/DA/Ledger/Services/PartyManagementService.hs

```diff
@@ -37,6 +39,14 @@ data PartyDetails = PartyDetails
     , isLocal :: Bool
     } deriving (Eq,Ord,Show)
 
+instance A.FromJSON PartyDetails where
```

Comment:
Isn't there an easy way to derive this? If this is the only `FromJSON` instance we add in this library it doesn't really matter but if start adding more it might make sense to look into that.

Reply:
you can use generics to derive it, but the field names would need to match the actual field names in the JSON data object.

Reply:
and yes there will be more instances like this when I implement the other endpoints.